server/database/db_build.sql

```diff
@@ -0,0 +1,53 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users cascade;
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_name VARCHAR(100) NOT NULL,
+  guid UNIQUEIDENTIFIER PRIMARY KEY default NEWID() ,
+  phone VARCHAR(12),
+  birth_date DATE,
+  email VARCHAR(30),
+  university VARCHAR(30),
+  address VARCHAR(100),
+  role VARCHAR(30),
+  profession VARCHAR(30),
+  password VARCHAR(30),
+  email_activate Boolean,
+  phone_activate Boolean
+);
+
+CREATE TABLE events(
+    id SERIAL PRIMARY KEY NOT NULL,
+    guid INT,
+    title VARCHAR(100) NOT NULL,
+    category_id INT,
```

Comment:
where the Foreign Key like this
FOREIGN KEY (category_id) REFERENCES Students(category),